schemas/json/layout/layout.schema.v1.json

```
@@ -139,6 +139,7 @@
         { "if": {"properties": {"type": { "const": "Likert"}}}, "then": { "$ref": "#/definitions/radioAndCheckboxComponents"}},
         { "if": {"properties": {"type": { "const": "MultipleSelect"}}}, "then": { "$ref": "#/definitions/selectionComponents"}},
         { "if": {"properties": {"type": { "const": "NavigationButtons"}}}, "then": { "$ref": "#/definitions/navigationButtonsComponent"}},
+        { "if": {"properties": {"type": { "const": "navigationBarComponent"}}}, "then": { "$ref": "#/definitions/navigationBarComponent"}},
```

```suggestion
        { "if": {"properties": {"type": { "const": "NavigationBar"}}}, "then": { "$ref": "#/definitions/navigationBarComponent"}},
```
The component type is `NavigationBar` (you can see all the possible options from the definition of the `type` property above).